data/coalition/lunarium intro.txt

```diff
@@ -792,6 +792,7 @@ mission "House Bebliss 5-FW"
 	landing
 	to offer
 		has "House Bebliss 4-FW: done"
+		has "outfit: Jump Drive" 2
```

Comment:
```suggestion
		"outfit: Jump Drive" >= 2 
```

Reply:
Thanks, I forgot that it being a condition here isn't the same thing.